tensorflow/python/kernel_tests/reader_ops_test.py

```diff
@@ -858,5 +858,30 @@ def _RunSessionAndSave(sess, args, output):
     output.append(sess.run(args))
 
 
+class LMDBReaderTest(test.TestCase):
+
+  def setUp(self):
+    super(LMDBReaderTest, self).setUp()
+
+  def testReadSimpleKV(self):
+    with self.test_session() as sess:
+      reader = io_ops.LMDBReader(name="test_reader")
+      path = os.path.join("tensorflow", "core", "lib", "lmdb", "testdata",
+                          "simplekv.mdb")
+      queue = data_flow_ops.FIFOQueue(99, [dtypes.string], shapes=())
+      key, value = reader.read(queue)
+
+      queue.enqueue([path]).run()
+      queue.close().run()
+      for i in range(10):
+        k, v = sess.run([key, value])
+        self.assertEqual(k, str(i))
+        self.assertEqual(v, str(chr(ord('a') + i)))
+
+      with self.assertRaisesOpError("is closed and has insufficient elements "
+                                    "\\(requested 1, current size 0\\)"):
+        k, v = sess.run([key, value])
+
```

Comment:
When opening a database file, LMDB creates a lock file `lock.mdb` in the same folder to prevent that database being opened more than once. This unfortunately prevents us to enqueue the database file multiple times.